.github/workflows/notify-triager.yml

```diff
@@ -6,12 +6,28 @@ on:
 
 jobs:
   Notify-triagers:
-    if: ! contains(github.event.pull_request.head.sha, 'Merge branch')
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v4.1.1
 
+      - name: Get commit message
+        id: commit-message
+        run: |
+          commit_message=$(git log --format=%B -n 1 ${{ github.event.pull_request.head.sha }})
+          echo "Commit message: $commit_message" >> $GITHUB_OUTPUT
+
+      - name: Check if last commit is a merge commit
+        id: check-merge-branch
+        run: |
+          if [[ ${{ steps.commit-message.outputs.commit_message }} == *"Merge branch"* ]]; then
+            echo "Last commit is a merge commit"
+            echo "skip" >> $GITHUB_OUTPUT
+          else
+            echo "Last commit message does not contain Merge branch"
+            echo "proceed" >> $GITHUB_OUTPUT
+          fi
```

Comment:
Pretty sure it's a typo. Unless you want to write a `if` condition after `else` block? :P @akshatnema 

LGTM. 

```suggestion
 
```